src/global/global-styles.js

```diff
@@ -0,0 +1,186 @@
+import { StyleSheet, Dimensions } from "react-native";
+/**
+ * This is the style sheet for the log in/sign up forms.
+ */
+
+const containerWidth="100%";
+const imgHeight=100;
+const imgWidth=100;
+const btnFontSize=19;
+const inputFontSize=16;
+const letterSpacing=0.25;
+const borderRadius=100;
+const btnmarginBottom=10;
+const borderWidth=1;
+const navBtnPaddingLeftRight="6%";
+const navBtnPaddingTopBottom="3%";
```

Comment:
all these numbers should be in the global constant 

btnmarginBottom should be marginSmall